product_docs/docs/jdbc_connector/42.5.4.1/06_executing_sql_commands_with_executeUpdate().mdx

```diff
@@ -19,65 +19,95 @@ The signature of the `executeUpdate()` method is:
 ```
 
-Provide this method a single parameter of type `String` containing the SQL command that you want to execute.
+Provide this method with a single parameter of type `String` containing the SQL command that you want to execute. 
+
+!!! Warning "Avoid user-sourced values"
+    We recommend that this string does not contain any user-sourced values. Avoid concatenating strings and values to compose your SQL command.  Instead, use [PreparedStatements](#using-preparedstatements-to-send-sql-commands) which are reusable, parameterized SQL statements which safely manage the use of variable values in the SQL statement.
+
 
 ## Using executeUpdate() to INSERT data
 
 The example that follows shows using the `executeUpdate()` method to add a row to the `emp` table.
 
-!!! Note
-    The following example isn't a complete application, only a method. These code samples don't include the code required to set up and tear down a `Connection`. To experiment with the example, you must provide a class that invokes the sample code.
+!!! Note Code samples
+    The following examples are not a complete application, only example methods. These code samples don't include the code required to set up and tear down a `Connection`. To experiment with the example, you must provide a class that invokes the sample code.
 
 
 ```java
-    public void updateEmployee(Connection con)
+    public void addOneEmployee(Connection con)
     {
-      try
-      {
-        int rowcount = stmt.executeUpdate("INSERT INTO "
-          + "emp(empno, ename) VALUES(6000,'Jones')");
-        System.out.println("");
-        System.out.println("Success - "+rowcount+
-          " rows affected.");
-      } catch(Exception err) {
-        System.out.println("An error has occurred.");
-        System.out.println("See full details below.");
-        err.printStackTrace();
-      }
+        try (Statement stmt=con.createStatement();)
+        {
+            int rowcount = stmt.executeUpdate("INSERT INTO emp(empno, ename) VALUES(6000,'Jones')");
+            System.out.println();
+            System.out.printf("Success - %d - rows affected.\n",rowcount);
+        } catch(Exception err) {
+            System.out.println("An error has occurred.");
+            System.out.println("See full details below.");
+            err.printStackTrace();
+        }
     }
 ```
 
-The `updateEmployee()` method expects a single argument from the caller, a `Connection` object that must be connected to an EDB Postgres Advanced Server database:
+The `addOneEmployee()` method expects a single argument from the caller, a `Connection` object that must be connected to an EDB Postgres Advanced Server database:
 
 ```java
-    public void updateEmployee(Connection con);
+    public void addOneEmployee(Connection con);
+```
+
+A `Statement` object is needed to run `ExecuteUpdate()`. This can be obtained by using `createStatement()` on the Connection object. We use the try-resource style here to 
```

Comment:
```suggestion
A `Statement` object is needed to run `ExecuteUpdate()`. This can be obtained by using `createStatement()` on the Connection object. We use the try-resource style here to 
```
Is there some missing text in the last sentence?